### DISCUSSION TITLE
it/api-agenzia-entrate/

### DISCUSSION THREAD

**Discussion by giscus** [matteocontrini/blog#10] [Comments]
# it/api-agenzia-entrate/

L’Agenzia delle Entrate sta preparando delle nuove API per la verifica di partite IVA e dei codici fiscali, ma la fase di sperimentazione non sta andando benissimo.

https://matteosonoio.it/it/api-agenzia-entrate/

**Comment by AndreaDiCeglie** [comment]
Grazie Matteo per l'ottimo riassunto che coincide con le mie conclusioni e per le info in più (piani, autenticazione e throttling) a cui non ero riuscito ad accedere.

Ad ogni modo pare che dalla data del tuo articolo (Luglio? Ottobre?) non ci sia stata nessuna novità

**Reply by matteocontrini** [reply]
Ciao, purtroppo non risultano novità nemmeno a me. L'articolo è di ottobre 2023 e non è successo altro che io sappia. Attendiamo.

**Comment by marzobryant90** [comment]
Bell’articolo Matteo. Più passa il tempo e più leggo le stesse cose sulle api di AdE, ovvero che non esistono.
Tu hai qualche novità?

**Reply by matteocontrini** [reply]
Ciao, non ho più seguito più di tanto ma leggendo sul forum Italia.it non mi pare sia cambiato molto ed è ancora tutto molto confuso. Mi dispiace non poter aiutare ulteriormente.

**Comment by marzobryant90** [comment]
Io sarei stato interessato ad API più complesse, tipo per la fatturazione
elettronica o anche per le dichiarazioni 730, ma è come chiedere di vincere
stasera al superenalotto (senza giocare).
Ci sono società (tipo FiscoAPI) che vendono prodotti che usano API fornite
da AdE, ma io non capisco come facciano visto che non esiste nulla di
ufficiale.
Alcune società, credo che siano controllate dal ministero magari tramite
Sogei, per questo riescono ad offrire soluzioni simili.

Digitalmente siamo ancora al 1990.


[image: Claudio Marzorati]
Claudio Marzorati

Founder & IT Manager • MM DigiBeauty

+39 347 9815145 <+39+347+9815145>

***@***.***

via Livio 7, Sordevolo, Biella
<https://www.google.com.ua/maps/search/via+Livio+7,+Sordevolo,+Biella>




[image: Linkedin] <https://www.linkedin.com>

*https://www.mmdigibeauty.com <https://https://www.mmdigibeauty.com>*


Il giorno ven 26 set 2025 alle ore 15:43 Matteo Contrini <
***@***.***> ha scritto:
